index.html

```diff
@@ -400,6 +419,67 @@ <h2>
         initially set to <code>null</code>, which is a promise whose associated
         operation is to lock the screen orientation.
       </p>
+      <section>
+        <h2>
+          Screen orientation types and locks
+        </h2>
+        <p>
+          <dfn>Landscape-primary</dfn> is an orientation where the screen width
+          is greater than the screen height. If the device's natural
+          orientation is landscape, then it is in landscape-primary when held
+          in that position. If the device's natural orientation is portrait,
+          the <a>user agent</a> sets landscape-primary from the two options as
+          shown <a>screen orientation values table</a>.
+        </p>
+        <p>
+          <dfn>Landscape-secondary</dfn> is an orientation where the screen
+          width is greater than the screen height. If the device's natural
+          orientation is landscape, it is in landscape-secondary when rotated
+          180º from its natural orientation. If the device's natural
+          orientation is portrait, the <a>user agent</a> sets
+          landscape-secondary from the two options as shown in the <a>screen
+          orientation values table</a>.
+        </p>
+        <p>
+          <dfn>Portrait-primary</dfn> is an orientation where the screen width
+          is less than or equal to the screen height. If the device's natural
+          orientation is portrait, then it is in portrait-primary when held in
+          that position. If the device's natural orientation is landscape, the
+          <a>user agent</a> sets portrait-primary from the two options as shown
+          in the <a>screen orientation values table</a>.
+        </p>
+        <p>
+          <dfn>Portrait-secondary</dfn> is an orientation where the screen
+          width is less than or equal to the screen height. If the device's
+          natural orientation is portrait, then it is in portrait-secondary
+          when rotated 180º from its natural position. If the device's natural
+          orientation is landscape, the <a>user agent</a> sets
+          portrait-secondary from the two options as shown in the <a>screen
+          orientation values table</a>.
+        </p>
+        <p>
+          <dfn>portrait</dfn> is an orientation where the screen width is less
```

Comment:
Don’t forget to capitalize these ones too :)

Reply:
So sorry I missed those!  Will do